__tests__/db.test.js

```diff
@@ -0,0 +1,83 @@
+const data = require("../db/data/test-data")
+const app = require("../app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const request = require("supertest")
+
+beforeEach(() => seed(data))
+
+afterAll(() => {
+  db.end()
+})
+
+describe("Errors", () => {
+  test("404: Unknown url not found", () => {
+    return request(app).get("/api/toothpicks").expect(404)
+  })
```

Comment:
Do you expect a message when you get a request to a bad endpoint?

When you come to build the front of the API, it may be useful for your future users to receive some information back